tests/testthat/test-get_vpts.R

```diff
@@ -545,16 +578,9 @@ test_that("get_vpts() returns an error for a bad radar", {
       datetime = "2023-01-01",
       source = "uva"
     ),
-    class = "getRad_error_radar_not_found"
-  )
-  expect_identical(
-    rlang::catch_cnd(get_vpts(
```

Comment:
This call ends now it a quite cryptic error:
``` r
devtools::load_all("~/getRad")
#> ℹ Loading getRad
get_vpts(
       radar = c( "deess", "bezav"),
       datetime = "2023-01-01",
       source = "baltrad"
   )
#> Error in `purrr::map()`:
#> ℹ In index: 2.
#> Caused by error in `get_vpts_aloft()`:
#> ! No data found for the requested radar(s) and date(s).
```
It would be good to provide the user with some guidance were the error is, that was the idea behind having the missing radars in the error call. (here deess has data but bejab not if i'm right), what do you think?

Reply:
This is a bug, this error is only supposed to trigger when there is no data found at all. And there indeed is coverage for deess.

Reply:
This bug got introduced because I moved the error from `get_vpts()` to `get_vpts_aloft()`, querying the helper one radar at a time. Thus it fetches the radar that has coverage in one call, the next call it finds no data at all, and returns the error interrupting the evaluation. 

So either we refactor the helper to query multiple radars at a time (ie, loop internally instead of in the parent), or we adjust the error handling and `purrr::map` in `get_vpts()` to more elegantly forward the error. With `purrr::possibly()` for example.

Reply:
I think either works, maybe the second solution is easier in the future as less logic is needed in source specific helper functions. I find it harder though to asses how much is involved in implementing that solution